.github/workflows/test.yml

```
@@ -284,66 +265,12 @@ jobs:
-              echo "✅ Found build for $name at $BUILD_PATH"
-            fi
-          done < <(echo "$PACKAGES" | jq -c '.[]')
-
-          echo "needs-rebuild=$MISSING" >> $GITHUB_OUTPUT
-      - name: Rebuild on missing artifacts
-        if: steps.verify.outputs.needs-rebuild == '1'
-        run: |
-          echo "::warning::Some Cypress builds missing — rebuilding all"
-          npm run cypress:server:build:coverage -- --concurrency=4
+          mode: restore
+          packages: ${{ needs.Get-Cypress-Packages.outputs.packages }}
```

actionlint (1.7.12)

[error] property "get-cypress-packages" is not defined in object type {cypress-setup: {outputs: {}; result: string}; get-test-groups: {outputs: {test-groups: string}; result: string}; lint: {outputs: {}; result: string}; setup: {outputs: {}; result: string}; type-check: {outputs: {}; result: string}}

(expression)